examples/tutorials/ordinary_model_example.ipynb

```diff
@@ -0,0 +1,407 @@
+{
```

Comment:
<p>If you have a torch model as an instance of <code>torch.nn.Module</code> it can be easily integrated into a training or inference pipeline.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/383/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>